Initium-Odp/src/com/universeprojects/miniup/server/commands/CommandGroupMemberKick.java

```diff
@@ -67,7 +67,12 @@ public final void run(final Map<String, String> parameters)
 		if (!"Admin".equals(admin.getProperty("groupStatus")))
```

Comment:
As a convention preference, I prefer to use ==false because it can be easy to miss a little !. It also reads better. Readability is highly preferred.